src/index.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
```

Comment:
using a global styles file for things like `<body>`, `<html>` and importing things like normalize.scss etc. is completely valid - just remember to have only one such file for the entire project.